ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -57,20 +57,27 @@ message Commands {
   // Required
   repeated Command commands = 8;
 
+  // Specifies the deduplication period for the change ID.
+  // If omitted, the participant chooses a deduplication period at its discretion.
```

Comment:
This is somewhat at odds with what is now written for `deduplication_duration`.

Reply:
Replaced it with the description originally next to `deduplication_duration`.